src/Generator.Bind/BindingsWriter.cs

```diff
@@ -514,26 +527,22 @@ private void WriteEnums(EnumCollection enums, FunctionCollection wrappers)
                         WriteLicense(sw);
                         sw.WriteLineNoTabs();
 
-                        sw.WriteLine("using System;");
-                        sw.WriteLineNoTabs();
+                        if (@enum.IsFlagCollection || @enum.IsObsolete)
+                        {
+                            sw.WriteLine("using System;");
+                            sw.WriteLineNoTabs();
+                        }
 
+                        sw.WriteLine("// ReSharper disable InconsistentNaming");
                         sw.WriteLine("#pragma warning disable SA1139 // Use literal suffix notation instead of casting");
```

Comment:
Is this required? I tried looking for places which break this rule but couldn't find any.

Reply:
It's mainly for the enumerations and their members - it shows up as a bunch of warnings otherwise.

Reply:
Right, that's fine then - I did know what it was for, but couldn't find any examples where it would show warnings. Figured it couldn't hurt to ask